packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx

```diff
@@ -477,12 +477,14 @@ export function VerifyCertificatePage() {
                     <ListViewItemSimplified
                       label={
                         <Text variant={'bold16'} element={'span'}>
-                          {intl.formatMessage(messageToDefine.createdAt)}
+                          {intl.formatMessage(
+                            messageToDefine.dateOfCertification
+                          )}
                         </Text>
                       }
                       value={
                         <Text variant={'reg16'} element={'span'}>
-                          {formatDate(new Date(data.createdAt), 'dd MMMM yyyy')}
+                          {formatDate(new Date(), 'dd MMMM yyyy')}
```

Comment:
isn't this the page where you land after scanning the QR code on the certificate? so the certification can have happened in the past and isn't necessarily the current date

Reply:
Yeah. We have to put the last certified date here.

Reply:
Updated.